src/comments/dto/update-comment.dto.ts

```diff
@@ -0,0 +1,22 @@
+import { IsArray, IsNotEmpty, IsOptional, IsString } from 'class-validator';
+import { ApiProperty } from '@nestjs/swagger';
+
+export class UpdateCommentDto {
+  @ApiProperty()
+  @IsOptional()
+  @IsString()
+  @IsNotEmpty()
+  title?: string;
+
+  @ApiProperty()
```

Comment:
Shouldn't optional property be set to null? Swagger shows it as required.
```suggestion
  @ApiProperty({ nullable: true })
```

Same about title and links